src/app/components/source/SourceComponent.js

```diff
@@ -1054,7 +1053,7 @@ class SourceComponent extends Component {
 
           { !isEditing ?
             <div>
-              { this.state.showTab === 'annotation' ? <Annotations annotations={source.annotations.edges.slice().reverse()} annotated={source} annotatedType="Source" /> : null }
+              { this.state.showTab === 'annotation' ? <Annotations annotations={source.log.edges} annotated={this.props.source} annotatedType="ProjectSource" /> : null }
```

Comment:
Yes as I'd like to create annotation using  `ProjectSourceID` as `annotated_id` and `ProjectSource` as `annotated_type but need to list logs for sources.